### DISCUSSION TITLE
How to add custom css for markdown?

### DISCUSSION THREAD

**Discussion by mandip1213** [chipzoller/hugo-clarity#350] [General]
**I have checked all the prerequisites below and I'm yet experiencing a problem**

- [x] Read the README.md
- [x] Have the extended version of Hugo installed
- [x] Used the exampleSite's config.toml as a reference
- [ ] I have included my repository link. Instead of pasting long snippets here.

**Describe the bug**
Its not a bug but a feature query. How can I change the default css applied to the markdown, How can I do it without breaking the whole site. Where should I put the css code? And how can I change the default blue background color? Sorry if the question sounds too naive, I am just getting started with hugo.

> Avoid long-winded descriptions

**Desktop (please complete the following information):**
 - OS:  Windows 10
 - Browser: Microsoft edge

**Comment by chipzoller** [comment]
A discussion was better for this so I've converted it. I believe @onweru answered this is another issue. Have you searched?

**Reply by rootwork** [reply]
@mandip1213 This should answer your question: https://github.com/chipzoller/hugo-clarity/discussions/340#discussioncomment-3137304

**Reply by mandip1213** [reply]
@rootwork  As mentioned in the [here](https://github.com/chipzoller/hugo-clarity/discussions/340#discussioncomment-3137304), I created a new file `assets/sass/_variables.sass` in my project directory and override the color values but that doesnot seem to work.
[Here ](https://github.com/mandip1213/hugo-demo/blob/main/assets/sass/_variables.sass) is what I have done exactly. Just for testing, I have converted all colors to `#000000`.

**Reply by rootwork** [reply]
Hmm, @onweru Can you confirm that just overriding `assets/sass/_variables.sass` is sufficient?

For my local setup I use `assets/sass/_override.scss` to override components and add new ones and I know that works, but I haven't actually tried just overriding values in the variables file.

**Reply by onweru** [reply]
Unless something changed in Hugo latest releases, it used to be sufficient, as long as the paths and filename mirrored each other.

**Reply by onweru** [reply]
Retesting to confirm.

**Reply by onweru** [reply]
@rootwork, @mandip1213,

I just reproduced it on my local. So yeah, it should work. All i did is create a new `assets/sass/_variables.sass` from the site dir.

<img width="1381" alt="Screen Shot 2022-07-29 at 00 51 41" src="https://user-images.githubusercontent.com/16350351/181643695-15d25610-e08b-43a0-91d2-5b64d7aaa44b.png">

**Reply by onweru** [reply]
A tangential idea 🤔. We should probably move completely to using CSS variables (at least for all colors), to make working with that file easier. I think CSS variables now enjoy better support than when I first wrote the variables down.

**Reply by mandip1213** [reply]
@onweru  Can you share the code what you have done. [Here](https://github.com/mandip1213/hugo-demo/blob/main/assets/sass/_variables.sass) I tried to create my own `_variables.sass` but nothing changes in my case.

**Reply by onweru** [reply]
@mandip1213,

I copied your code to my local and it turns out you had errors on lines [97](https://github.com/mandip1213/hugo-demo/blob/a1b15f5db5f4ac00a9b2ff16cb9d6a9c765ae30e/assets/sass/_variables.sass#L97) and [121](https://github.com/mandip1213/hugo-demo/blob/a1b15f5db5f4ac00a9b2ff16cb9d6a9c765ae30e/assets/sass/_variables.sass#L121). This way, hugo was erroring out. Here is how the correct code for those lines should look like 👇🏼 

```sass
// line 97
background-image: url("#{imagesPath}night-moon.jpg");
// line 121
background-image: url("#{imagesPath}night-moon.jpg");
```
